magento-engcom/community-portal
# magento-engcom/community-portal#77 Only users with privileges should see the edit and create project buttons

magento-engcom/community-portal#77 Only users with privileges should see the edit and create project buttons

- [x] Enforce checks on related project endpoints for users to have write:project scope
- [x] Write tests to make sure that scope checking works correctly
- [x] Hide the create/edit button and enforce checks before showing the buttons

Testing
- Added two users, one with write:project scope and the other without write:project scope

Test cases
- Check that the user with write:project scope can create project and the other cannot
- Check that the user with write:project scope can edit project and the other cannot
- Check that the user with write:project scope can update project status and the other cannot
